sp800-63-3/sec3_definitions.md

```diff
@@ -62,7 +62,7 @@ In this document, biometrics may be used to unlock authenticators and prevent re
 A party whose identity is to be verified using one or more authentication protocols.
 
 #### Claimed Identity
-A declaration by the applicant of their current Personal Name, date of birth and address. [[GPG45]](#gpg45)
+A declaration of unvalidated and unverified personal attributes by the applicant.
```

Comment:
This definition should also appear in -63A, where the term is used extensively. There is currently no definition for *claimed identity* there.